gh-actions-scripts/cleanup_docker_images.sh

```diff
@@ -62,27 +93,23 @@ for s in ${IMAGES[@]}; do
   # get all outdated tag where tag contains "bug"
   outdated_bug_tags=$(get_outdated_images $s "bug" $MAX_AGE)
   # get all outdated tag where tag contains "patch"
-  outdated_patch_tags=$(get_outdated_images $s "patch" $MAX_AGE)
+  # outdated_patch_tags=$(get_outdated_images $s "patch" $MAX_AGE)
```

Comment:
is this line disabled intentionally?

Reply:
good catch. Yes it's disabled intentionally, but I should also disable the for loop below.